### ISSUE TITLE
Remove Text Domain line if 4.6+

### ISSUE THREAD

**Issue by ePascalC** [ePascalC/CheckPluginForTranslation#22]
ERROR: Your plugin slug is XXX, but your Text Domain is XXXX. Change your Text Domain in X.php so it is equal to your slug!

Indicate at the end: OR just delete the Text Domain line because your required version is 4.6+ MORE INFO

**Comment by ePascalC** [comment]
If text domain is set in file:
  If Required version is <4.6:
    ERROR: Your plugin slug is XXX, but your Text Domain is XXXX. Change the Text Domain tag in X.php (and modify the text domain in all your source files) so it is equal to your slug! This change is needed because your Required version is below 4.6. MORE INFO
  If 4.6+:
    ERROR: Your plugin slug is XXX, but your Text Domain is XXXX. Change the Text Domain tag in X.php (and modify the text domain in all your source files) so it is equal to your slug! As the Required Version is at least 4.6, you could also remove the Text Domain tag, but you still need to modify all your source files. MORE INFO

If text domain not set:
  If Required version is <4.6:
    ERROR: There is no Text Domain tag found in X.php. Please add it and set it equal to your plugin slug. This change is needed because your Required version is below 4.6. MORE INFO
  If Required version is 4.6+:
    Assume plugin slug

**Comment by pbiron** [comment]
I just updated my [Show Environment in Admin Bar](https://wordpress.org/plugins/show-environment-in-admin-bar) plugin as we discussed earlier today on slack.

The page for my plugin on [translate.wordpress.org](https://translate.wordpress.org/projects/wp-plugins/show-environment-in-admin-bar) is no longer showing the "This plugin is not properly prepared for localization."  Yeah!

However, [this tool](http://wp-info.org/tools/checkplugini18n.php?slug=show-environment-in-admin-bar) is still reporting an error:

> ERROR: Your plugin slug is show-environment-in-admin-bar, but there seems no Text Domain: defined in plugin.php! To correctly internationlize your plugin you will need it.

that I think is related to this issue.
